docs/changes/io.fits/12808.feature.rst

```diff
@@ -0,0 +1,3 @@
+Add option ``unit_format`` to ``astropy.io.fits.connect.read_table_fits`` to
+enable using a wider range of units for files not compliant with the units
+defined in the FITS Standard.
```

Comment:
Is it worth mentioning this can be used inside `Table.read(fits_file)`? Maybe
```
Add option ``unit_format`` to ``astropy.io.fits.connect.read_table_fits`` 
(and thus to ``Table.read(format='fits')``) to enable using a wider
range of units for files not compliant with the units defined in the
FITS Standard.
```

Reply:
Done.